src/data/ACTIONS/root/PLD.ts

```diff
@@ -304,4 +328,28 @@ export const PLD = ensureActions({
 		onGcd: false,
 		cooldown: 30000,
 	},
+	IMPERATOR: {
+		id: 36921,
+		name: 'Imperator',
+		icon: iconUrl(2955),
+		onGcd: false,
+		cooldown: 60000,
+		statusesApplied: ['REQUIESCAT', 'CONFITEOR_READY'],
+	},
+	BLADE_OF_HONOR: {
+		id: 36922,
+		name: 'Blade of Honor',
+		icon: iconUrl(2956),
+		onGcd: false,
+		// Setting a cooldown is technically incorrect, but xivanalysis core modules require it for it to show up on the timeline
+		cooldown: 30000,
```

Comment:
Hey! I'm a fellow new arrival here (showed up the other day to look at adding warrior stuff for 7.0.), so not qualified to review properly. But about this section:

> The second workaround regards to Blade of Honor. As far as I'm aware it does not have a cooldown, since it is tied to the Blade of Honor Ready buff. Not giving it a cooldown, however will result in the skill not being shown on the timeline, so I just gave it a cooldown of 30s.

I think the correct cooldown to set for this would be 1000 (ie 1 second), since that's what's listed in the job guide as the recast time. It's effectively never going to be relevant since you can't gain the Blade of Honor Ready status more than once a minute, but setting that would matches what the tooltip says.

Reply:
> I think the correct cooldown to set for this would be 1000 (ie 1 second), since that's what's listed in the job guide as the recast time. It's effectively never going to be relevant since you can't gain the Blade of Honor Ready status more than once a minute, but setting that would matches what the tooltip says.

Good point! Totally missed that. Adjusted accordingly.

Reply:
@kwerey Would you be so kind to resolve this conversation, if the change is in line with your expectations. :)